rust/README.md

```diff
@@ -0,0 +1,42 @@
+# Rust on CodeQL
+
```

Comment:
I'd put a warning about this being experimental and unsupported, just in case

Reply:
Done.